sunpy/io/fits.py

```diff
@@ -23,6 +23,7 @@
    invalid header tags will result in an error so verifying it early on
    makes the header easier to work with later.
 """
+import math
```

Comment:
Just need to move this to between sys and warnings.

Reply:
That's not how the alphabet works 🤣